base/replutil.jl

```diff
@@ -197,7 +197,6 @@ end
 
 function showerror(io::IO, ex::TypeError)
     print(io, "TypeError: ")
-    ctx = isempty(ex.context) ? "" : "in $(ex.context), "
     if ex.expected === Bool
         print(io, "non-boolean ($(typeof(ex.got))) used in boolean context")
```

Comment:
err sorry yeah nevermind, I misread that as if `ctx` was getting used in this but it clearly isn't